docs/decisions/0003-static-builds-to-client-side-react.rst

```diff
@@ -0,0 +1,55 @@
+
+Taking the course detail pages as an example, let's assume a worst-case scenario where we have 500 Enterprise Customers, each with a course catalog that grants access to all edX content (i.e., ~2500 courses). In this scenario, because each course detail page has branding that is unique to each individual Enterprise Customer, we would potentially need to build 1,250,000 static pages to support the course catalogs of those 500 Enterprise Customers. This would lead to long build times that would slow down development, testing, and rolling out production bug fixes.
+
+An argument might be made that we could implement a hybrid approach, where we only build each course page once and dynamically fetch the Enterprise Customer's data (e.g., logo, name, branding) client-side. However, at that point, we would not really be taking full advantage of Gatsby as a static site generator. Resources online point to scenarios such as this, where most of an application uses dynamic data, as rationale for not using Gatsby (see References).
+
+********
+Decision
+********
+
+While it would be great if the Enterprise learner portal could take advantage of the benefits for using Gatsby (e.g., faster page loads), it does not make as much sense given the tradeoffs mentioned above. This is because the Enterprise learner portal relies mostly on dynamic data in such a way that would not allow us to realize the advantage of a statically rendered app. Additionally, the Enterprise learner portal does not have a requirement for supporting SEO best practices since it is used by existing customers (presumably with direct access to the URL) as opposed to a marketing site that will be crawled by search engine bots.
+
+That said, we will be migrating the Enterprise learner portal off of Gatsby and its multi-site build process / infrastructure. Instead, the Enterprise learner portal will become a standard client-side React application (microfrontend) that utilizes many of the recent advancements in the frontend ecosystem at edX (e.g., ``@edx/frontend-build``, ``@edx/frontend-platform``).
+
+************
+Consequences
+************
+
+* Rather than sourcing the necessary configuration data during the build process, data for the Enterprise learner portal will be fetched client-side as needed from the browser.
+    * That said, we will want to be intentional about how we do client-side data fetching to reduce the amount of perceived loading time for the user.
+* The Enterprise learner portal will no longer use multi-site build process / infrastructure. Rather, the Enterprise learner portal will use the same build process and infrastructure as other standard microfrontends at edX.
+* The Enterprise learner portal will begin using ``@edx/frontend-build`` and ``@edx/frontend-platform`` to be in line with other microfrontends at edX and reduce common boilerplate.
+* Ownership of the ``gatsby-source-portal-designer`` Gatsby plugin will be transitioned to the Programs engineering team(s).
```

Comment:
I would add that ownership of https://github.com/edx/frontend-learner-portal-base was also transitioned to the programs team